backend/app/main.py

```
@@ -153,6 +153,19 @@ async def continue_chat(
     if not consultation:
         raise HTTPException(status_code=404, detail="相談が見つかりません")
     
+    if message:
+        if len(message) > 0:
+            if character_name:
+                if consultation_id:
+                    if tone:
```

Ruff (0.8.2)

Use a single `if` statement instead of nested `if` statements

Combine `if` statements using `and`

(SIM102)